styleguide/source/_patterns/00-atoms/02-text/18-file-size/file-size.md

```diff
@@ -0,0 +1,8 @@
+---
+el: ".file_size"
+title: "File size"
+---
+
+A file size atom created for use in Topic Tools.
```

Comment:
Same note as above about whether we describe the atom as coupled to the Topics pages.